client/lecturer/app/comment.panel.directive.js

```diff
@@ -25,7 +25,9 @@
     /* @ngInject */
     function CommentPanelController(lecturesFactory, $interval, $scope) {
         console.log('Ready (Comment Panel Controller)');
-
+        
+        $scope.sortByName = "-score";
```

Comment:
I think you should add this property to commentPanel (below) in stead of $scope (right, @121052 ?) Then in the html you say commentPanel.sortByName.